nasa/earthdata-search
# EDSC-3219: Allow user to disable temporal subsetting for Harmony orders

# Overview

### What is the feature?

Allows a user to disable temporal subsetting for Harmony orders

### What is the Solution?

Added a checkbox that disables temporal subsetting

### What areas of the application does this impact?

Project page

# Testing

### Reproduction steps

- **Environment for testing:** UAT
- **Collection to test with:** C1234724470-POCLOUD

1. Add collection to project
2. Disable temporal subsetting
3. Confirm `time` parameter is not sent to Harmony and order is not subset

# Checklist
